src/pages/Scheduling/components/CreateScheduleTemplateSheet.tsx

```diff
@@ -235,6 +236,34 @@ export default function CreateScheduleTemplateSheet({
     );
   };
 
+  const validFrom = form.watch("valid_from");
+  const validTill = form.watch("valid_to");
+  const [dateValidationCallout, setDateValidationCallout] =
+    useState<ReactNode>(null);
+
+  // show warning while filling "valid from" and "valid till" are in past
+  useEffect(() => {
+    if (!validFrom || !validTill) {
+      setDateValidationCallout(null);
+      return;
+    }
+
+    // Normalizing today’s date (removes time)
+    const today = startOfDay(new Date());
```

Comment:
I know Rithvik suggested using date-fns, but let's switch over to dayjs (we recently started moving over).